packages/react-component-library/src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useRef } from 'react'
+import classNames from 'classnames'
+
+import { POPOVER_CLOSE_DELAY } from './constants'
+
+import {
+  FloatingBox,
+  FloatingBoxProps,
+  FLOATING_BOX_SCHEME,
+  FLOATING_BOX_PLACEMENT,
+  FLOATING_BOX_PLACEMENT_ARROW_POSITION_MAP,
+} from '../../primitives/FloatingBox'
+
+import { calculate } from './popoverPosition'
+
+interface PopoverProps
+  extends Omit<FloatingBoxProps, 'onMouseEnter' | 'onMouseLeave'> {
+  children: React.ReactElement
+  popoverJSX: React.ReactElement
```

Comment:
I feel like a nicer name for this could be `content` or something similar.

Reply:
maybe `renderTarget`